app/keepers/keepers.go

```diff
@@ -273,7 +273,6 @@ func NewAppKeeper(
 		appKeepers.StakingKeeper,
 		appKeepers.SlashingKeeper,
 		appKeepers.AccountKeeper,
-		appKeepers.EvidenceKeeper,
```

Comment:
This is bc the equivocation proposal was removed in ICS `v2.2.0-provider-lsm-rc0`